lib/types.ts

```diff
@@ -1744,6 +1745,22 @@ export type QuickReplyItem = {
   action: Action;
 };
 
+export type Sender = {
+  /**
+   * Display name
+   *
+   * - Max: 20 characters
+   * - Certain words such as LINE may not be used.
+   */
+  name?: string;
+  /**
+   * Icon image URL (Max: 1000 characters)
+   *
+   * - **HTTPS**
+   */
+  iconUrl?: string;
```

Comment:
Would you please change the comments to the following one? I will be very appreciated.
```
/**
* URL of the image to display as an icon when sending a message
*
* - Max character limit: 1000
* - URL scheme: https
*/
```